templates/docker-compose.yml.j2

```
@@ -34,12 +35,18 @@ services:
       test: ["CMD", "wget", "-qO-", "http://127.0.0.1:{{ nim_waku_metrics_port }}/health"]
       interval: 30s
       retries: 3
+    entrypoint: /opt/entrypoint.sh
     command: |
       --config-file=/conf/config.toml
-{# WARNING: Keep sensitive information in Docker Compose file. #}
+{# WARNING: Keep sensitive information in run-node.sh. #}
```

nitpick comment. On the other hand, it seems that we are keeping sensitive info in both the script and the docker-compose.yml file. I think is interesting to keep it in the script file only, to simplify it a little.
```suggestion
{# WARNING: Keep sensitive information in entrypoint.sh. #}
```